setup.py

```diff
@@ -16,7 +16,7 @@ def get_requirements():
 
 setup(
     name='ixbrl_viewer',
-    version='0.0.0',
+    version='@VERSION@',
```

Comment:
Why change the placeholder?  That seems unrelated to the issue described in the ticket, which is that the line number is specific.  (It might be good to include the relevant detail in the PR description given this is an open source repo.)